src/AppInstallerCommonCore/Archive.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+
+namespace AppInstaller::Archive
+{
+    HRESULT ExtractArchive(const std::filesystem::path& archivePath, const std::filesystem::path& destPath)
+    {
+        IFileOperation* pFileOperation;

```

Comment:
Every one of these COM pointers should be `wil::com_ptr<Interface>`.

Reply:
Changed to using `wil::com_ptr<Interface>`